style.css

```
@@ -0,0 +1,30 @@
+body {
+    font-family: "Roboto", "Arial", sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 20px;
+    background-color: #849d8f;
+    color: white;
```

```suggestion
html {
    box-sizing: border-box;
}
*, *::before, *::after {
    box-sizing: inherit;
    margin: 0;
    padding: 0;
}

body {
    font-family: "Roboto", "Arial", sans-serif;
    font-size: 16px;
    font-weight: 400;
    line-height: 20px;
    background-color: #849d8f;
    color: white;
```

тупа всегда добавляй это наверх и будешь на 90% меньше мучаться с цссом